xain/grpc/conftest.py

```diff
@@ -39,3 +39,15 @@ def participant_stub():
     stub = coordinator_pb2_grpc.CoordinatorStub(channel)
 
     return stub
+
+
+# similar to coordinator_service but yields the coordinator servicer
```

Comment:
Instead of a new fixture can't we just change `coordinator_service`? It shouldn't break existing tests.